chromadb/api/__init__.py

```diff
@@ -129,6 +132,7 @@ def get_or_create_collection(
         self,
         name: str,
         metadata: Optional[CollectionMetadata] = None,
+        configurable: Optional[Configurable] = None,
```

Comment:
I think we want the inputs here to be strongly typed, so that its very clear to users what the options are.